policyengine_us_data/datasets/cps/cps.py

```diff
@@ -1446,19 +1449,6 @@ def get_arrival_year_midpoint(peinusyr):
     )
     immigration_status[mask] = "TPS"
 
-    # ---------------------------------------------------------------
-    # Map custom labels into Enum-approved buckets
-    placeholder_to_specific_status = {
```

Comment:
There were no TEMP_NONQUALIFIED statuses before this, so removing this code has no effect